### PR TITLE
Bump to SpringBoot@2.6.2

### PR DESCRIPTION
**Pull request by mcornaton**
SpringFox has been replaced by SpringDoc. Let me know if the new UI suits you!

### CODE REVIEW

**Comment by zguesmi** [comment]
For mongo properties conflict issue, how about keeping everything as it was before and replace this line
```
registry.add("spring.data.mongodb.uri", mongoDBContainer::getReplicaSetUrl);
```
by
```
registry.add("spring.data.mongodb.host", mongoDBContainer::getContainerIpAddress);
registry.add("spring.data.mongodb.port", () -> mongoDBContainer.getMappedPort(27017));
```
?

**Comment by mcornaton** [comment]
@zguesmi I guess it works. I was sure I had already tried this easy solution, maybe in an old version of Spring Boot. Thanks!

File: src/main/java/com/iexec/core/stdout/StdoutCronService.java [hunk 1]
```diff
@@ -47,11 +44,9 @@ public StdoutCronService(
         this.taskService = taskService;
     }
 
-    public long getPurgeRateInMs() {
-        return TimeUnit.DAYS.toMillis(purgeRateInDays);
-    }
-
-    @Scheduled(fixedRateString = "#{@stdoutCronService.getPurgeRateInMs()}")
+    @Scheduled(
```
Comment (zguesmi) [inline src/main/java/com/iexec/core/stdout/StdoutCronService.java:47]: :broom:

**Review by zguesmi — APPROVED** [review]
Neat!

**Review by jeremyjams — APPROVED** [review]
Welldone!

**Pull request merged**